advanced/pattern-1/apim/wso2apim-1-service.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the license header year to latest, since this is a new file.

Applies to all new files.

Reply:
done